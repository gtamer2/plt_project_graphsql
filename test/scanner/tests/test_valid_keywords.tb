AND
OR
DEFINE
FUNCTION
CREATE 
SELECT
FROM 
AS 
WHERE
INSERT
UNION
INTERSECT
APPLY
GRAPH
VERTEX
EDGE
NOT
vertices
edges
WHILE
IF
ELSE
ELIF