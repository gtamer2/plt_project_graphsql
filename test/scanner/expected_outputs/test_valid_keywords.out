AND
OR
DEFINE
FUNCTION
CREATE
SELECT
FROM
AS
WHERE
INSERT
UNION
INTERSECT
APPLY
GRAPH
VERTEX
EDGE
NOT
VERTICES
EDGES
WHILE
IF
ELSE
ELIF